test_dataset.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+import pandas as pd
+
+
+ALLOWED_COLUMN_NAMES = [
+    # record_id
+    "record_id",
+
+    # default metadata
+    "title",
+    "abstract",
+    "keywords",
+    "authors",
+    "year",
+    "date",
+    "doi",
+
+    # custom variables
+    "label_included",
+    "label_abstract_screening",
+    "duplicate_record_id",
+
+    # special cases
+    "pubmedID"
```

Comment:
Why only pubmed?

Reply:
This is the only ID in the dataset so far. I'm not happy with it, but not sure what the best solution is. The ideal solution would be record_id containing PubMedIDs, however, I'm not sure how to communicate this with the user.